docs/csharp/tour-of-csharp/program-building-blocks.md

```diff
@@ -291,6 +233,67 @@ The garbage collector is allowed wide latitude in deciding when to collect objec
+
+For the complete list of C# operators ordered by precedence level, see [C# operators](../language-reference/operators/index.md).
+
+## Statements
+
+The actions of a program are expressed using *statements*. C# supports several different kinds of statements, a number of which are defined in terms of embedded statements.
+
+- A *block* permits multiple statements to be written in contexts where a single statement is allowed. A block consists of a list of statements written between the delimiters `{` and `}`.
+- *Declaration statements* are used to declare local variables and constants.
+- *Expression statements* are used to evaluate expressions. Expressions that can be used as statements include method invocations, object allocations using the `new` operator, assignments using `=` and the compound assignment operators, increment and decrement operations using the `++` and `--` operators and `await` expressions.
+- *Selection statements* are used to select one of a number of possible statements for execution based on the value of some expression. This group contains the `if` and `switch` statements.
+- *Iteration statements* are used to execute repeatedly an embedded statement. This group contains the `while`, `do`, `for`, and `foreach` statements.
+- *Jump statements* are used to transfer control. This group contains the `break`, `continue`, `goto`, `throw`, `return`, and `yield` statements.
+- The `try`...`catch` statement is used to catch exceptions that occur during execution of a block, and the `try`...`finally` statement is used to specify finalization code that is always executed, whether an exception occurred or not.
+- The `checked` and `unchecked` statements are used to control the overflow-checking context for integral-type arithmetic operations and conversions.
+- The `lock` statement is used to obtain the mutual-exclusion lock for a given object, execute a statement, and then release the lock.
+- The `using` statement is used to obtain a resource, execute a statement, and then dispose of that resource.
+
+The following lists the kinds of statements that can be used:
+
+* Local variable declaration.
+* Local constant declaration.
+* Expression statement.
+* `if` statement.
+* `switch` statement.
+* `while` statement.
+* `do` statement.
+* `for` statement.
+* `foreach` statement.
+* `break` statement.
+* `continue` statement.
+* `goto` statement.
+* `return` statement.
+* `yield` statement.
+* `throw` statements and `try` statements.
+* `checked` and `unchecked` statements.
+* `lock` statement.
+* `using` statement.
+
+When a method body is a single expression, the method can be defined using a compact expression format, as shown in the following example:
```

Comment:
This fits more the section about methods (how to concisely define methods), not the one about statements.